src/lib.rs

```diff
@@ -74,10 +74,9 @@ pub trait ThreadTracer {
 }
 
 /// Start tracing on the current thread using the specified tracing kind.
-/// By passing `kind` as `None` an appropriate tracing kind will be selected automatically. A
-/// thread tracer is returned through which tracing can be stopped and the trace obtained. The user
-/// must ensure that only one thread tracer exists at any one time per-thread, otherwise undefined
-/// behaviour will result.
+/// If `None` is passed, then an appropriate tracing kind will be selected; by passing `Some(...)`,
+/// a specific kind can be chosen. Tracing different threads concurrently is allowed, but tracing
```

Comment:
The second sentence doesn't really help IMHO. Maybe something like "Any given thread can only have one active tracer; calling `start_tracing` on a thread where there is already an active tracer leads to undefined behaviour".

Reply:
I like it!

Reply:
"can only have one active tracer" might better be "must have at most one active tracer"?

Reply:
I wrote exactly the same independently!